aquality-appium-mobile-template/src/main/java/aquality/appium/mobile/template/screens/ios/Alert.java

```diff
@@ -2,23 +2,24 @@
 
 import aquality.appium.mobile.screens.IOSScreen;
 import aquality.appium.mobile.template.screens.interfaces.IAlert;
-import org.apache.commons.lang3.NotImplementedException;
 
-import static io.appium.java_client.MobileBy.AccessibilityId;
+import static io.appium.java_client.MobileBy.*;
 
 public class Alert extends IOSScreen implements IAlert {
 
     public Alert() {
-        super(AccessibilityId("not_implemented"), "Alert");
+        super(iOSNsPredicateString("type == 'XCUIElementTypeAlert' AND name == 'Alert'"), "Alert");
     }
 
     @Override
     public String getMessage() {
-        throw new NotImplementedException("iOS screens are not implemented");
+        return getElementFactory()
+                .getLabel(iOSClassChain("**/XCUIElementTypeOther[$name == 'Alert'$]/XCUIElementTypeStaticText[2]"), "Message")
+                .getText();
     }
 
     @Override
     public void tapOk() {
-        throw new NotImplementedException("iOS screens are not implemented");
+        getElementFactory().getButton(AccessibilityId("OK"), "OK").click();
```

Comment:
please save elements into private final fields to make element caching work

Reply:
fixed